https/tls_version.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build go1.12
+
+package https
+
+import (
+	"crypto/tls"
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+func pickMinVersion(s string) (uint16, error) {
+	switch strings.ToUpper(s) {
+	case "TLS1.3":
```

Comment:
TLS13 is what Go uses, and I presume at some point it'll get iterators like the ciphers have so let's make that future transition easier.

Reply:
updated